src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+class Node<T> {
+    T data;
+    Node<T> next;
+    Node<T> prev;
+
+    Node(T data) {
+        this.data = data;
+        this.next = null;
+        this.prev = null;
+    }
+}
```

Comment:
Consider moving the `Node` class as an inner class within `MyLinkedList` to encapsulate it properly. This will also allow you to make the fields private, improving encapsulation.